### PR TITLE
chore: change license to GNU GPLv3

### PR DESCRIPTION
**Pull request by immadisairaj**
Change license from MIT to GNU GPLv3

### CODE REVIEW

**Review by vibhoothi — COMMENTED** [review]
Why ?

File: LICENSE [hunk 1]
```diff
@@ -1,21 +1,674 @@
-MIT License
-
-Copyright (c) 2019 FOSS@Amrita
```
Comment (vibhoothi) [inline LICENSE:3]: If you even making it as GNUv3 you cannot remove
`Copyright (c) 2019 FOSS@Amrita`

**Review by vibhoothi — CHANGES_REQUESTED** [review]
If you are going to merge this, at least please squash the commits

File: LICENSE (hunk 1 above)
Comment (immadisairaj) [reply LICENSE:3]: now fixed

**Review by vibhoothi — CHANGES_REQUESTED** [review]
I would also like to have a one-line description stating why we changed it in commit message itself, so it will not look shady.

**Comment by immadisairaj** [comment]
> I would also like to have a one-line description stating why we changed it in commit message itself, so it will not look shady.

yeah sure

**Review by vibhoothi — APPROVED** [review]

**Pull request merged**